PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp

```diff
@@ -918,7 +940,7 @@ LRESULT DockingCont::runProcTab(HWND hwnd, UINT Message, WPARAM wParam, LPARAM l
 						tcItem.mask		= TCIF_PARAM;
 						::SendMessage(hwnd, TCM_GETITEM, iItem, reinterpret_cast<LPARAM>(&tcItem));
 						if (!tcItem.lParam)
-							return FALSE;
+							return 1;

```

Comment:
In online document of WM_MOUSMOVE:
https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-mousemove
say:
```
Return value
If an application processes this message, it should return zero.
```
So IMO it should return 0 anyway.

Reply:
This should be `break;` instead, to pass it to default window processing.